Assembly Tutorial/projects/com.vantiq.weatherEmergencies.WeatherEmergencyProject.json

```diff
@@ -0,0 +1,325 @@
+  }, {
+    "source" : "procedure/com.vantiq.weatherEmergencies.DetectWeatherEmergency.com_vantiq_weatherEmergencies_isDangerousWeather_getTemperatureAndWind",
+    "target" : "services/com.vantiq.weatherEmergencies.DetectWeatherEmergency"
+  }, {
+    "source" : "procedure/com.vantiq.weatherEmergencies.DetectWeatherEmergency.com_vantiq_weatherEmergencies_isDangerousWeather_WarnMessage",
+    "target" : "services/com.vantiq.weatherEmergencies.DetectWeatherEmergency"
+  }, {
+    "source" : "procedure/com.vantiq.weatherEmergencies.DetectWeatherEmergency.com_vantiq_weatherEmergencies_isDangerousWeather_EvacuationMessage",
+    "target" : "services/com.vantiq.weatherEmergencies.DetectWeatherEmergency"
+  }, {
+    "source" : "services/com.vantiq.weatherEmergencies.DetectWeatherEmergency",
+    "target" : "type/com.vantiq.weatherEmergencies.weatherReading"
+  }, {
+    "source" : "services/com.vantiq.weatherEmergencies.DetectWeatherEmergency",
+    "target" : "app/com.vantiq.weatherEmergencies.DetectWeatherEmergency_WeatherEvents_EventHandler"
+  }, {
+    "source" : "app/com.vantiq.weatherEmergencies.DetectWeatherEmergency_WeatherEvents_EventHandler",
+    "target" : "app/com.vantiq.weatherEmergencies.isDangerousWeather"
+  }, {
+    "source" : "app/com.vantiq.weatherEmergencies.DetectWeatherEmergency_WeatherEvents_EventHandler",
+    "target" : "services/com.vantiq.weatherEmergencies.DetectWeatherEmergency"
+  }, {
+    "source" : "procedure/com.vantiq.weatherEmergencies.DetectWeatherEmergency_WeatherEvents_EventHandler.com_vantiq_weatherEmergencies_isDangerousWeather_EvacuationMessage",
+    "target" : "services/com.vantiq.weatherEmergencies.DetectWeatherEmergency"
+  }, {
+    "source" : "procedure/com.vantiq.weatherEmergencies.DetectWeatherEmergency_WeatherEvents_EventHandler.com_vantiq_weatherEmergencies_isDangerousWeather_WarnMessage",
+    "target" : "services/com.vantiq.weatherEmergencies.DetectWeatherEmergency"
+  }, {
+    "source" : "procedure/com.vantiq.weatherEmergencies.DetectWeatherEmergency_WeatherEvents_EventHandler.com_vantiq_weatherEmergencies_isDangerousWeather_getTemperatureAndWind",
+    "target" : "services/com.vantiq.weatherEmergencies.DetectWeatherEmergency"
+  } ],
+  "name" : "com.vantiq.weatherEmergencies.WeatherEmergencyProject",
+  "options" : {
+    "description" : "This is the Assembly Tutorial. The weather Source produces temperature and wind speed data which triggers the DetectWeatherEmergencyApp. The App uses the isDangerousWeather App Component to determine whether to issue a warning message or evacuation alert to residents. Each alert type will trigger the start of a CollaborationType which will notify the ResidentNotificationSystem Client.\n\nThe Assembly exposes the App Component used by the DetectWeatherEmergency App, the activation status for the weather Source, and the theme on the ResidentNotificationSystem Client as Configurable Properties.",
```

Comment:
if the CollaborationTypes have been removed, this the last line of the first paragraph needs to be updated here.

Reply:
and in the README.md, too.